src/translations/sqlb_de.ts

```diff
@@ -5171,15 +5171,15 @@ Grund: %1</translation>
     </message>
     <message>
         <source>Duplicate records</source>
-        <translation type="vanished">Doppelte Einträge</translation>
+        <translation type="vanished">Doppeltee Datensätze</translation>
```

Comment:
Please, note that German is not my native language, but isn't it here a mistake in the second added "e" in "Doppeltee"?

Reply:
absolute! Thx